src/core/stream.h

```diff
@@ -433,7 +432,10 @@ typedef struct QUIC_STREAM {
 
     //
     // The number of received bytes the app has completed but not yet processed
-    // by MsQuic.
+    // by MsQuic. The top bit of RecvCompletionLength is used to indicate that
```

Comment:
nit: If using the second top-most bit as a canary, we should update the comment with it too.